centosplus patch

Ref: ELRepo patch elrepo-mpt3sas-unremove-el8.patch

Fixes:
https://elrepo.org/bugs/view.php?id=919

Some device IDs are listed as 'support_removed' in RHEL kernels.
Do not return error when unsupported hardware is detected.

It probably makes sense to leave the check itself in place.

--- a/drivers/scsi/mpt3sas/mpt3sas_scsih.c	2018-08-31 19:51:40.686480022 +0200
+++ b/drivers/scsi/mpt3sas/mpt3sas_scsih.c	2018-09-01 19:04:46.634532270 +0200
@@ -10583,7 +10583,7 @@
 
 	if (pci_device_support_removed(mpt3sas_pci_table,
 				mpt3sas_pci_ids_removed, pdev))
-		return -ENODEV;
+		printk(KERN_DEBUG "mpt3sas: elrepo version supporting all pci ids\n");
 
 	/* Determine in which MPI version class this pci device belongs */
 	hba_mpi_version = _scsih_determine_hba_mpi_version(pdev);
